mypy/build.py

```diff
@@ -1748,7 +1748,7 @@ def check_blockers(self) -> None:
             self.manager.errors.raise_error()
 
     @contextlib.contextmanager
-    def wrap_context(self) -> Iterator[None]:
+    def wrap_context(self, check_blockers: bool = True) -> Iterator[None]:
```

Comment:
I would add a comment with motivation for this. My understanding is that if a blocker occurs in top-level, we still need to process functions, to maintain backwards compatibility with the old analyzer.